src/wwwroot/template-editor/adapterService.js

```diff
@@ -71,6 +70,56 @@
       $window.location = "/";
     }
 
+    /**
+     * Return the Image Gallery
+     * @argument {Object} params - Configuration object
+     * @argument {string} params.query - The file name and extension
+     * @argument {string} params.position - The provider name
+     * @argument {number} params.offset - The campaign identifier
+     * @argument {number} params.sortingCriteria - The template identifier
+     * @argument {number} params.isAscending - The template identifier
+     * @example
+     * ```js
+     * var params = {
+     *   fileName: 'testFile.jpg',
+     *   providerName: 'Youtube',
+     *   idCampaign: 1,
+     *   idTemplate: 2
+     * };
+     * 
+     * adapterService.getImagesForCampaign(params);
+     * ```
+     */
+    function getImagesForCampaign(start, howMany, query, sortingCriteria) {
```

Comment:
Could you move this block of code to the previous position to see the diff more clearly?

![imagen](https://user-images.githubusercontent.com/1157864/44680489-3e887c00-aa14-11e8-920e-c8273021eb35.png)